### PR TITLE
Monday code interfaces

### PR DESCRIPTION
**Pull request by AvnerMaster**
Added the structure for monday code interfaces

### CODE REVIEW

**Review by oricho123 — APPROVED** [review]
Great work! left some comments

File: quickstart-node/src/types/monday-code.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+export interface MondayCodeSecretManager {
+  get: (key: string, options?: GetKeyOptions) => string;
+}
+
+export interface MondayCodeEnvironmentVariablesManager {
+  get: (key: string, options?: GetKeyOptions) => string;
+}
+
+export interface GetKeyOptions {
+  invalidate?: boolean;
```
Comment (oricho123) [inline quickstart-node/src/types/monday-code.ts:29]: What is the use of this?

Comment (AvnerMaster) [reply quickstart-node/src/types/monday-code.ts:29]: for cached results

File: quickstart-node/src/types/monday-code.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+export interface MondayCode {
```
[the remark this answers is not in the document]
Comment (AvnerMaster) [reply quickstart-node/src/types/monday-code.ts:1]: Good point, to be honest, i'm not sure, it depends on the needs, if we want flexibility than this type is unnecessary, but if we want to enforce using these utilities exactly, than it is necessary. What do you think?

File: quickstart-node/src/types/monday-code.ts
[the remark this answers is not in the document]
Comment (AvnerMaster) [reply quickstart-node/src/types/monday-code.ts:1]: Done, do you also want me to change the interface name from MondayCode to MondayCodeSdk?

[the remark this answers is not in the document]
Comment (oricho123) [reply quickstart-node/src/types/monday-code.ts:1]: Yes!

File: quickstart-node/src/types/monday-code.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (oricho123) [reply quickstart-node/src/types/monday-code.ts]: Maybe we can find some middle point to enforce using utilities but not being as strict.
I think that the services should be independent and not under MondayCode class

File: quickstart-node/src/types/monday-code.ts (hunk 1 above)
Comment (oricho123) [reply quickstart-node/src/types/monday-code.ts]: Great!

**Pull request merged**